borg/selftest.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import time
+from unittest import TestResult, TestSuite, defaultTestLoader
```

Comment:
hmm, do we want to depend on unittest for this?
i rather thought we would like to get rid of it and replace everything by py.test.
but ofc we would not depend on py.test for production.